readme.md

```diff
@@ -50,10 +50,10 @@ Passing `Infinity` will cause it to never time out.
 
 ##### message
```

Comment:
What do you think about renaming this to `error`? I think `error: false` would read better.

Reply:
Agree that. At this point `error: false` is good enough that we don't need a specific `throwOnTimeout` prop